packages/ui/src/components/sheet/sheet.native.tsx

```diff
@@ -1,83 +1,24 @@
 import { ReactNode, forwardRef } from 'react';
-import { ViewStyle } from 'react-native';
-import Animated, {
-  Extrapolation,
-  SharedValue,
-  interpolate,
-  useAnimatedProps,
-  useAnimatedStyle,
-  useSharedValue,
-} from 'react-native-reanimated';
+import { SharedValue, useSharedValue } from 'react-native-reanimated';
 import { useSafeAreaInsets } from 'react-native-safe-area-context';
 
 import {
-  BottomSheetBackdropProps,
   BottomSheetModal,
   BottomSheetScrollView,
   BottomSheetTextInput,
   BottomSheetView,
-  useBottomSheet,
 } from '@gorhom/bottom-sheet';
 import { useTheme } from '@shopify/restyle';
 
-import { Box, Theme, TouchableOpacity, createTextInput } from '@leather.io/ui/native';
+import { Box, Theme, createTextInput } from '@leather.io/ui/native';
 
-import { ThemedBlurView } from './blur-view';
+import { ThemeVariant } from '../../theme-native/theme';
+import { Backdrop } from './components/sheet-backdrop.native';
 
 export const UIBottomSheetTextInput = createTextInput(BottomSheetTextInput);
 
-const absoluteStyle = {
-  position: 'absolute',
-  top: 0,
-  bottom: 0,
-  right: 0,
-  left: 0,
-} satisfies ViewStyle;
-
-const AnimatedBlurView = Animated.createAnimatedComponent(ThemedBlurView);
-
-function Backdrop({ animatedIndex }: BottomSheetBackdropProps) {
-  const { close } = useBottomSheet();
-
-  const animatedStyle = useAnimatedStyle(() => ({
-    opacity: interpolate(animatedIndex.value, [-1, -0.5], [0, 0.3], Extrapolation.CLAMP),
-  }));
-
-  const animatedProps = useAnimatedProps(() => {
-    const intensity = interpolate(animatedIndex.value, [-1, -0.5], [0, 15], Extrapolation.CLAMP);
-    return {
-      intensity,
-    };
-  });
-
-  return (
-    <AnimatedBlurView
-      animatedProps={animatedProps}
-      style={[
-        {
-          width: '100%',
-          justifyContent: 'center',
-        },
-        absoluteStyle,
-      ]}
-    >
-      <Animated.View
-        style={[
-          {
-            backgroundColor: 'black',
-          },
-          absoluteStyle,
-          animatedStyle,
-        ]}
-      >
-        <TouchableOpacity onPress={() => close()} style={{ flex: 1 }} />
-      </Animated.View>
-    </AnimatedBlurView>
-  );
-}
 export const CLOSED_ANIMATED_SHARED_VALUE = -888;
-
-export const Modal = forwardRef<
+export const Sheet = forwardRef<
```

Comment:
@edgarkhanzadian I have more things to update but is this more on the right track? 

I've moved `Modal` and renamed it as `Sheet`